Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -338,8 +337,7 @@ class BrowserViewController: UIViewController {
         }
 
         header.snp_remakeConstraints { make in
-            let topLayoutGuide = self.topLayoutGuide as! UIView
-            scrollController.headerTopConstraint = make.top.equalTo(topLayoutGuide.snp_bottom).constraint
+            scrollController.headerTopConstraint = make.top.equalTo(self.topLayoutGuide).constraint
```

Comment:
This one is a bit tricky too since SnapKit doesn't support UILayoutSupport's bottomAnchorSupport property yet. Not sure what to do here except for set the constraint to the view itself and account for the offset by taking into account the length of the topLayoutGuide. This is probably why we're seeing the toolbar at the top of the screen. We fix this in another PR/patch.